build.gradle

```diff
@@ -249,7 +251,7 @@ task jacocoRootReport(type: org.gradle.testing.jacoco.tasks.JacocoReport) {
         html.enabled = true
         xml.enabled = true
         csv.enabled = false
-        xml.destination = "${buildDir}/reports/jacoco/test/jacocoTestReport.xml"
+        //xml.destination = "${buildDir}/reports/jacoco/test/jacocoTestReport.xml"
```

Comment:
same here ?

Reply:
Uncommented and fixed the compilation against last Gradle, thanks for notifying